taggit/managers.py

```diff
@@ -142,9 +142,15 @@ def uuid_rel_obj_attr(v):
     def _lookup_kwargs(self):
         return self.through.lookup_kwargs(self.instance)
 
+    def _remove_prefetched_objects(self):
+        try:
+            self.instance._prefetched_objects_cache.pop(self.prefetch_cache_name)
+        except (AttributeError, KeyError):
+            pass  # nothing to clear from cache
```

Comment:
stylewise, I think it would be nice if this was structured more like:

```
obj_cache = getattr(self.instance, "_prefetched_objects_cache", None)
if obj_cache:
    obj_cache.pop(self.prefetch_cache_name, None)
```

That way we don't have a `try`/`except` catching `AttributeError` and `KeyError` (both pretty heavily used exceptions). 

I don't think the code here is wrong but I think my proposed alternative makes it clearer what the expected situations are (for example I don't think we are expecting `self.prefetch_cache_name` to through an `AttributeError`)